src/style.css

```
@@ -1 +1,74 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  color: #000;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  list-style: none;
+  text-decoration: none;
+}
+
+.link:hover,
+.is-active {
```

you already have a class that combines the two

```suggestion
.item-link:hover {
```